modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -2362,7 +2362,9 @@ def broadcast_apply_full_axis(
             result.synchronize_labels(axis=1)
         return result
 
-    def _copartition(self, axis, other, how, sort, force_repartition=False):
+    def _copartition(
+        self, axis, other, how, sort, force_repartition=False, make_reindex=True
```

Comment:
```suggestion
        self, axis, other, how, sort, force_repartition=False, perform_reindex=True
```
you don't "make" an operation so it is applied, you "perform", "apply", "do" or "execute" it 🙃